src/models/photos.model.js

```diff
@@ -0,0 +1,37 @@
+const mongoose = require("mongoose");
+
+const photoSchema = new mongoose.Schema({
+  info: {
+    type: [
+      {
+        url: {
+          type: String,
+        },
+        createdBy: {
+          type: String,
+        },
+        likes: {
+          type: [
+            {
+              type: String,
+            },
+          ],
+        },
+        tags: {
+          type: [
+            {
+              type: String,
+            },
+          ],
+        },
+        location: {
+          type: String,

```

Comment:
Me hace sentido que las fotos tengan ubicaciones. Pero guardar todo en un string esta complicado. Mas si quieres filtrar despues. (esto va para cualquier tipo de location en cualquier model.
```suggestion
          type: {
             city: String,
             state: String,
             country: String,
             zipcode: String,
          },
          
```

Reply:
Tienes razón, cuando estaba haciendo el schema vi que lo necesitabamos pero ya no lo desarrollé bien. Ya lo cambié